tests/compiletest.rs

```diff
@@ -87,7 +128,7 @@ fn main() {
     // Panic tests expect backtraces to be printed.
     env::set_var("RUST_BACKTRACE", "1");
```

Comment:
Lol, so we set it to "1" here and then reset it to "0" in the ui_test library? :joy: 
Can we just remove the `set_var` here? Avoiding that API would be better anyway.